test/java/seedu/addressbook/commands/ViewCommandTest.java

```diff
@@ -74,8 +74,8 @@ public void execute_validIndex_returnsPersonDetails() {
         // person with all private information
         assertViewSuccess(typicalAddressBook, listWithAllTypicalPersons, 4);
 
-        // addressbook has more people than the list
-        // This can happen when view from filtered list caused by some commands(eg. FindCommand)
+        // addressbook has more people than the list.
+        // this can happen when a command causes the list to show only a sub-set of persons(e.g. FindCommand).
```

Comment:
Full sentences should start with a upper case letter too. :-) So it should either a full sentence starting with a upper case letter and ending with a full stop, or just a short phrase starting with a lower case letter and no full stop at the end.